docs/source/index.rst

```diff
@@ -1,118 +1,136 @@
+* Perform incident response on partially ransomwared (encrypted) virtual machine disks
+* Perform complex IOC checks on thousands of hosts in a couple of hours
+* Export all USN journal records from a Bitlocker-encrypted disk to Splunk without waiting for decryption
+* Collect forensic artefacts from all live virtual machines directly from the hypervisor with no down time
+* Export all artefacts easily to any data format you want, for example CSV, JSON or Avro, or stream directly to Splunk or Elastic
+
+
+Easy to setup
+-------------
+
+Dissect can be installed using pip:
 
 .. code-block:: console
 
-    $ pip install dissect.ntfs dissect.regf
+    $ pip install dissect
+
+It is recommended that you use a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_.
+
+.. note ::
+    .. include:: /versions.rst
+    
+You can also try dissect directly from within your browser:
 
-You can find an overview of all available Dissect projects and their documentation at :doc:`/projects/index`.
+`try.dissect.tools <https://try.dissect.tools>`_
+    
 
-Using the Docker image
-~~~~~~~~~~~~~~~~~~~~~~
+Setup with Docker
+-----------------
 
 If you simply want to get started with some of the examples without having to install anything, a basic Docker image
 is available `here <https://github.com/orgs/fox-it/packages/container/package/dissect>`_ (`mirror <https://hub.docker.com/r/dissect/dissect>`_).
 You can start using this image by executing the following command in your terminal:
-
+ 
 .. code-block:: console
 
-    $ docker run -it --rm -v /path/to/targets/:/mnt:ro ghcr.io/fox-it/dissect:3.2
+    $ docker run -it --rm -v /path/to/targets/:/mnt:ro ghcr.io/fox-it/dissect:3.11
     (<dissect version>) <container hash>:/workspace$
 
-This will drop you in a shell environment with all the Dissect ``target-*`` :doc:`tools </tools/index>` at your disposal.
-The purpose of the ``-v`` option is to mount a local directory inside the Docker container. You can use this to interact
-with local data from within the Docker container.
 
-Browser demo
-~~~~~~~~~~~~
+Learn more
+----------
 
-If you don't feel like doing the above, then there is an interactive browser demo available at
-`try.dissect.tools <https://try.dissect.tools>`_ to play around with!
+Excited? Great, so are we!
+The next step is to get to know Dissect a little better.
 
-You can select evidence files from your local system, such as VMDK or EWF files, and explore some of the capabilities
-of Dissect. Nothing gets uploaded to a server, it all happens locally thanks to `Pyodide <https://pyodide.org/en/stable/>`_!
+* Visit :doc:`/usage/introduction` for an in-depth introduction into Dissect
+* Visit :doc:`/tools/index` for an overview of each tool in the Dissect suite
 
-.. caution::
+Or you can start by taking a look at some community articles and videos:
+:doc:`/resources/dissect-in-action` or
+:doc:`/resources/talks-and-conferences` to begin with.
```

Comment:
Adding a little bit of how to get in touch with us on this page might be a good idea. (or linking to it somewhere here).